packages/lib/gpc/README.md

```diff
@@ -34,26 +34,26 @@
 
 A library for creating and verifying zero-knowledge proofs using General Purpose
 Circuits. For a full introduction, see the
-[Developer Site](https://zupass.org/pod-developers).
+[Developer Site](https://zupass.org/pod).
 
-**POD** is a format enabling any app to flexibly create cryptographic data and
-make zero-knowledge proofs about it. A POD could represent your ticket to an
-event, a secure message, a collectible badge, or an item in a role-playing game.
-Using PODs, developers can create ZK-enabled apps without the effort and risk of
-developing their own cryptography.
+**POD** libraries enable any app to create zero-knowledge proofs of cryptographic data. A POD could represent your ticket to an event, a secure
+message, a collectible badge, or an item in a role-playing game. Using PODs,
+developers can create ZK-enabled apps without the effort and risk of developing
+custom cryptography.
 
 ZK proofs about PODs use General Purpose Circuits (**GPC**) which can prove many
-different things about a POD without revealing it all. GPCs use human-readable
-configuration and pre-compiled circuits so no knowledge of circuit programming
-is required.
+different things about PODs without revealing all details. GPCs use
+human-readable configuration and pre-compiled circuits so no knowledge of
+circuit programming is required.
 
 PODs and GPCs can be used in Zupass, or in your own apps without Zupass.
 
 ## What is a GPC?
 
-GPCs allow ZK proofs to be created from a simple proof configuration. You can
-configure your proofs using a human-readable (JSON) format, which is used to
-generate the specific circuit inputs needed for the proof.
+PCs allow ZK proofs to be created from a simple proof configuration. You can
```

Comment:
Unless this is just a confusing diff, the "G" in "GPC" is missing in the first sentence.